test/powershell/Scripting/NativeExecution/NativeCommandArguments.Tests.ps1

```diff
@@ -0,0 +1,22 @@
+Describe "Native Command Arguments" -tags "CI" {
+    $testPath = 'TestDrive:\test.txt'
+    $powershellTestDir = $PSScriptRoot
+    while ($powershellTestDir -notmatch 'test[\\/]powershell$') {
+        $powershellTestDir = Split-Path $powershellTestDir
+    }
+    $echoArgs = Join-Path (Split-Path $powershellTestDir) tools/EchoArgs/bin/echoargs
+
+    It "Should handle quoted spaces correctly" {
+        $a = 'a"b c"d'
+        & $echoArgs $a 'a"b c"d' a"b c"d >$testPath
+        $testPath | Should Contain 'Arg 0 is <ab cd>'
+        $testPath | Should Contain 'Arg 1 is <ab cd>'
+        $testPath | Should Contain 'Arg 2 is <ab cd>'
```

Comment:
`Should Contain` usage here makes it's harder to troubleshoot the error. When it accured.
Here, I'm trying to execute the test on the inbox powershell v5.1

```
Describing Native Command Arguments
 [+] Should handle quoted spaces correctly 619ms
 [-] Should handle spaces between escaped quotes 127ms
   Expected: file TestDrive:\test.txt to contain {Arg 0 is <a"b c"d>}
   19:         $testPath | Should Contain 'Arg 0 is <a"b c"d>'
   at <ScriptBlock>, F:\dev\PowerShell\test\powershell\Scripting\NativeExecution\NativeCommandArguments.Tests.ps1: line 19
```

As you see, I don't have information about the produced outpuit immediately in my disposal.
Just to find what we are currently produce I would need to run commands by hands.

I suggest to replace `$testPath | Should Contain 'Arg 0 is <ab cd>'` by

```powershell
$lines = Get-Contetxt $testPath
($lines | measure).Count | Should Be 3  # this line will help get a better error message in case there is only one line in the output
$lines[0] | Should Be 'Arg 0 is <ab cd>'
```